content/en/docs/contribute/generate-ref-docs/contribute-upstream.md

```diff
@@ -53,24 +53,25 @@ creating a patch to fix it in the upstream project.
 
 ## Cloning the Kubernetes repository
 
-If you don't already have the kubernetes/kubernetes repository, get it now:
+Make sure your `kubernetes` fork is up-to-date with the `kubernetes/kubernetes` remote on
+GitHub (`master` branch), and clone your `kubernetes` fork.
 
 ```shell
 mkdir $GOPATH/src
 cd $GOPATH/src
-go get github.com/kubernetes/kubernetes
+git clone git@github.com:<your_github_username>/kubernetes.git
 ```
 
 Determine the base directory of your clone of the
 [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) repository.
 For example, if you followed the preceding step to get the repository, your
-base directory is `$GOPATH/src/github.com/kubernetes/kubernetes.`
+base directory is `$GOPATH/src/kubernetes.`
 The remaining steps refer to your base directory as `<k8s-base>`.
 
 Determine the base directory of your clone of the
 [kubernetes-sigs/reference-docs](https://github.com/kubernetes-sigs/reference-docs) repository.
 For example, if you followed the preceding step to get the repository, your
-base directory is `$GOPATH/src/github.com/kubernetes-sigs/reference-docs.`
+base directory is `$GOPATH/src/kubernetes-sigs/reference-docs.`
```

Comment:
This is not right either.

Reply:
Do we have a previous step for cloning this repo?

Reply:
@tengqm, There is no reference of 'kubernetes-sigs/reference-docs' above anywhere, it is only mentioned here.

Reply:
That means the instructions here are broken.
Go ahead and fix it, please.